docs/tarball.md

```diff
@@ -12,14 +12,23 @@ oci_tarball(
 )
 ```
 
-and then run it in a container like so:
+and then load and run it in a container like so:
 
 ```
-bazel build //path/to:tarball
-docker load --input $(bazel cquery --output=files //path/to:tarball)
-docker run --rm my-repository:latest
+bazel run :tarball
+docker run --rm "my-repository:latest"
```

Comment:
I think the original FR wanted to run a container from the image too, not just load it into the daemon. But it's really questionable - I was always confused in rules_docker whether a given target was the language kind or not, and the default behavior differed. (as much as it was fun to `bazel run -- --norun` just like you "analyze" with `bazel build --nobuild`

Also you've already fixed the main ergonomic problem (the `bazel cquery`)

Pretty much convincing myself this is already fine but I wonder if you have any thoughts.

Reply:
Given most people are coming from rules_docker, I thought it made sense for the behavior to be the same but @thesayyn suggested we didn't want that behavior so I removed it.

I was going to add an example to the ticket of extending oci_tarball with macro that would run the container as well in case it was wanted.

Open to other suggestions.

Reply:
we should not get into the business of running containers with a single command. it makes the API surface for oci_tarball more confusing. originally it was designed to be an `oci_image -> docker tarball` converter but we decided that it's common enough to get its `runnable` target which `loads` into the daemon as well. and we should stop there, this is already expanding the surface, which will make it harder for us to support other tarball formats like oci layout tarball.

Reply:
people coming from rules_docker are already used to both behaviors anyhow

https://github.com/bazelbuild/rules_docker#using-with-docker-locally

>For the lang_image targets, this will also run the container using docker run to maximize compatibility with lang_binary rules.

But plain `container_image` targets have the behavior we'll have here to just load the image with `bazel run`